catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/CatalogFrameworkImpl.java

```diff
@@ -1488,24 +1488,23 @@ protected ResourceResponse getResource(ResourceRequest resourceRequest, boolean
                 resourceSourceName = resolvedSourceId;
             }
 
-            String key;
-            try {
-                key = new CacheKey(metacard, resourceRequest).generateKey();
-            } catch (Exception e1) {
-                LOGGER.error("resource not found", e1);
-                throw new ResourceNotFoundException(e1);
+            String key = new CacheKey(metacard, resourceRequest).generateKey();
+            if (key == null) {
```

Comment:
As far as I can tell, this will never be true and we've now lost this "Resource not found" exception.